core/src/test/resources/newlines/alwaysBeforeTopLevelStatements.stat

```diff
@@ -29,25 +35,29 @@ package a {
   import x2.b
 
   package b {
-
     object a {
       val v1 = 1
       val v2 = 2
 
+      val v3 = {
+        3
+      }
+
       case class C1
 
       sealed trait T1
 
       final class FC1
```

Comment:
I don't mean to feature creep your PR, so feel free to de-scope this remark, but ideally (for me) these single line definitions wouldn't be force-split apart by the tool.